app/src/main/java/org/groundplatform/android/data/remote/firebase/schema/TaskConverter.kt

```diff
@@ -74,6 +74,13 @@ internal object TaskConverter {
         } else {
           null
         }
+
+      // Determine if user can move the point or if it's locked to device location
+      val allowMovingPoint = getAllowMovingPoint(taskType, task)
+
+      // Determine if user can manually override location lock for draw area tasks
+      val allowManualOverride = getAllowManualOverride(taskType, task)
```

Comment:
Iiuc, this splits the condition into two, `allowManualOverride` for area, `allowMovingPoint` for point. Since we're calling both cases "manual location override", can we merge these into one condition, `allowManualOverride`?